client/user/src/mealorder/menu/meal-order-user-cart.html

```diff
@@ -0,0 +1,179 @@
+  -->
+<script src="../../../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+<link rel="import" href="../../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../../bower_components/iron-list/iron-list.html">
+<link rel="import" href="../../../bower_components/paper-item/paper-item.html">
+<link rel="import" href="../../../bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../../../bower_components/paper-button/paper-button.html">
+<link rel="import" href="../../../bower_components/iron-icons/iron-icons.html">
+<link rel="import" href="../../../bower_components/app-layout/app-drawer-layout/app-drawer-layout.html">
+<link rel="import" href="../../../bower_components/app-layout/app-drawer/app-drawer.html">
+<link rel="import" href="../../../bower_components/app-layout/app-header/app-header.html">
+<link rel="import" href="../../../bower_components/app-layout/app-header-layout/app-header-layout.html">
+<link rel="import" href="../../../bower_components/app-layout/app-scroll-effects/app-scroll-effects.html">
+<link rel="import" href="../../../bower_components/app-layout/app-toolbar/app-toolbar.html">
+<link rel="import" href="../../../bower_components/iron-flex-layout/iron-flex-layout.html">
+<dom-module id="meal-order-user-cart">
+    <template>
+        <style>
+            :host {
+                color: #4f4f4f;
+            }
+
+            .order-details {
+                display: flex;
+                font-size: 15px;
+                align-items: center;
+                border-bottom: 2px solid #fcfcfc;
+            }
+
+            .vendor {
+                display: flex;
+                justify-content: space-between;
+                align-items: center;
+                padding-left: 4%;
+                background-color: #fcfcfc;
+            }
+
+            .cart {
+                overflow-x: hidden;
+                overflow-y: auto;
+                max-height: 93vh;
+            }
+
+            .order-title {
+                width: 50%;
+                text-align: left;
+                padding-left: 4%;
+            }
+
+            .order-price {
+                width: 20%;
+                text-align: right;
+            }
+
+            .order-quantity {
+                width: 15%;
+            }
+
+            .amount-title {
+                width: 50%;
+                text-align: left;
+                padding-left: 4%;
+                font-weight: bolder;
+            }
+
+            .amount-order-price {
+                width: 20%;
+                text-align: right;
+
+            }
+
+            .total {
+                display: flex;
+                justify-content: space-between;
+                margin: 6% 4% 10% 4%;
+                font-weight: bolder;
+            }
+        </style>
+        <div class="cart">
+
+            <dom-repeat items="[[order]]" as="dish">
```

Comment:
It's better to move this to upper component. Just create a component to render order from vendor.